packages/cardano-services-client/src/HandleProvider/KoraLabsHandleProvider.ts

```diff
@@ -26,17 +26,6 @@ export interface KoraLabsHandleProviderDeps {
   policyId: Cardano.PolicyId;
 }
 
-export interface HandleResolution {
-  policyId: Cardano.PolicyId;
-  handle: Handle;
-  cardanoAddress: Cardano.PaymentAddress;
-  hasDatum: boolean;
-  defaultForStakeKey?: Handle;
-  defaultForPaymentKey?: Handle;
-  backgroundImage?: Asset.Uri;
-  profilePic?: Asset.Uri;
-}
```

Comment:
Since `HandleResolution` is imported, you might need to remove `resolvedAt` [here](https://github.com/input-output-hk/cardano-js-sdk/blob/be395075e628dfeac02363f95e0a86e6b6bf3204/packages/core/src/Provider/HandleProvider/types.ts#L21)

Reply:
`resolvedAt` is optional, unless this requirements have changed @rhyslbw .